install/kubernetes/istio-auth-with-cluster-ca.yaml

```diff
@@ -126,7 +126,7 @@ spec:
       serviceAccountName: istio-pilot-service-account
       containers:
       - name: discovery
-        image: docker.io/istio/pilot:0.2.1
+        image: gcr.io/istio-testing/pilot:kuat_170901_Sep091504329418
```

Comment:
why can't that be something built by the CI ? even off your PR, don't we build even on presubmit ?
also you merged pilot 1160 so can you put the built artefact instead ?

Reply:
this is a two repo problem. now that 1160 got merged, I'll use it.

Reply:
yes, please use b5d710d658d3bfffa06506f312725d96a8737703